serverless.yml

```diff
@@ -566,3 +570,67 @@ functions:
               - name: request.querystring.graph_type
               - name: request.querystring.max_level
               - name: request.header.origin
+
+  walletServiceAddressBalance:
+    handler: handlers/wallet_service.handle_address_balance
+    timeout: 15
+    package:
+      patterns:
+        - 'handlers/wallet_service.py'
+    layers:
+      - { Ref: PythonRequirementsLambdaLayer }
+    events:
+      - http:
+          path: address/balance
+          method: get
+          cors: true
+          caching:
+            enabled: true
+            ttlInSeconds: 5
+            cacheKeyParameters:
+              - name: request.querystring.address
+              - name: request.querystring.token
+              - name: request.header.origin
+
+  walletServiceAddressHistory:
+    handler: handlers/wallet_service.handle_address_history
+    timeout: 15
+    package:
+      patterns:
+        - 'handlers/wallet_service.py'
+    layers:
+      - { Ref: PythonRequirementsLambdaLayer }
+    events:
+      - http:
+          path: address/history
+          method: get
+          cors: true
+          caching:
+            enabled: true
+            ttlInSeconds: 5
+            cacheKeyParameters:
+              - name: request.querystring.address
+              - name: request.querystring.token
+              - name: request.querystring.limit
+              - name: request.querystring.skip
```

Comment:
I think `offset` should be here instead of `skip`

Reply:
Fixed